src/layers/icon-layer/icon-layer.js

```diff
@@ -257,7 +257,7 @@ export default class IconLayer extends Layer {
 
     const layerProps = {
       radiusScale,
-      ...(this.config.visConfig.fixedRadius ? {} : {radiusMaxPixels: 500})
+      ...(this.config.visConfig.fixedRadius ? {} : {radiusMaxPixels: 15})
```

Comment:
the max radius only controls the maximum a point can be, why do you want to change it?

Reply:
Please check this: https://github.com/keplergl/kepler.gl/issues/1352